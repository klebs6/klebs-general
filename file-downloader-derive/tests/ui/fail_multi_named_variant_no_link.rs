
// file-downloader-derive/tests/ui/fail_multi_named_variant_no_link.rs
use file_downloader::FileDownloader;
use file_downloader_derive::FileDownloader;

#[derive(FileDownloader)]
enum MyEnum {
    MultiField {
        a: u32,
        b: u32,
    },
}

fn main() {}

