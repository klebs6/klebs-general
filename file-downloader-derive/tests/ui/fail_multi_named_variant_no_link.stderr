error: Multiple named fields not supported without a download_link
  --> tests/ui/fail_multi_named_variant_no_link.rs:8:5
   |
8  | /     MultiField {
9  | |         a: u32,
10 | |         b: u32,
11 | |     },
   | |_____^

error: `match` arm with no body
  --> tests/ui/fail_multi_named_variant_no_link.rs:8:5
   |
8  | /     MultiField {
9  | |         a: u32,
10 | |         b: u32,
11 | |     },
   | |     ^- help: add a body after the pattern: `=> todo!(),`
   | |_____|
   |
